packages/odyssey/src/scss/components/_checkbox.scss

```diff
@@ -14,15 +14,26 @@
 
 /* stylelint-disable no-descending-specificity, max-nesting-depth */
 
+$size-checkbox: 1.1428571429em;
```

Comment:
😑

Reply:
I believe we would want to use `rem` units here so that the layout will adjust if a lower sighted user has configured their browser with a larger default browser size.